Maple2.File.Ingest/Mapper/ItemMapper.cs

```diff
@@ -70,6 +72,73 @@ public class ItemMapper : TypeMapper<ItemMetadata> {
                 expirationDuration = data.life.usePeriod;
             }
 
+            var hairList = new List<DefaultHairMetadata>();
+            // parse default hair positions
+            foreach (Slot slot in data.slots.slot.Where(dataSlots => dataSlots.name == "HR")) {
+
+                // not sure what the difference/significance is within the multiple scale entries. Currently just using the first one
+                float minScale = slot.scale.ElementAtOrDefault(0)?.min ?? 0f;
+                float maxScale = slot.scale.ElementAtOrDefault(0)?.max ?? 0f;
+                switch (slot.asset.Count) {
+                    case 3: // Hair has front and back positionable hair section
+                        var backHairPositions3 = new Vector3[slot.asset[1].custom.Count];
+                        var backHairRotations3 = new Vector3[slot.asset[1].custom.Count];
+                        var frontHairPositions3 = new Vector3[slot.asset[2].custom.Count];
+                        var frontHairRotations3 = new Vector3[slot.asset[2].custom.Count];
+                        
+                        for (int index = 0; index < slot.asset[1].custom.Count; index++) {
+                            // back hair
+                            foreach (var preset in slot.asset[1].custom) {
```

Comment:
I'm a bit confused by the logic here, you are overwriting `backHairPositions3[index]` repeatedly for each preset. (you are only keeping track of the last preset since `index` is constant for this entire loop) Is this intentional?